docs/spec/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Why is this here? I went into this directory and typed `make venv` but it complained about a missing `conf.py`.

Reply:
Oops, probably a holdover from when it was in its own repo. I'll get rid of it. (You should `make` in the parent directory, which includes this spec and the other typing docs.)

Reply:
Yeah, I figured that out eventually. It's a bit weird how clicking on the Table of Contents header at the top of the sidebar takes you to the global ToC, not the Spec ToC, even though there is no "Next topic" linked from spec/historical.html, the last topic in Spec (so I would expect that the ToC link would go to the Spec ToC). But that's a very minor detail at this point, and probably some Sphinx oddity.